platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -99,7 +127,11 @@ class HangingProtocolService {
 
     // if there is no pre-defined protocol
     if (!protocol || protocol.id === undefined) {
-      const matchedProtocol = this.ProtocolEngine.run(metaData);
+      const matchedProtocol = this.ProtocolEngine.run({
+        studies: this.studies,
+        studyMetaData: studies[0],
+        displaySets,
```

Comment:
also, if this is active study displaysets, it should be renamed

Reply:
It is all the display sets associated with the studies.  There might be a concept of inactive display sets, but I'm not quite sure what it means - it is NOT active in terms of what is actually being shown right now, because that hasn't yet been determined.

Reply:
I don't think we use inactive display sets at the moment. It was originally envisioned to be there for cases when you switched modes within a session. Some display sets would be created for one mode but not relevant for another. We have previously discussed nuking it though since it's not being used and I don't think we need it.